doc/docusaurus.config.ts

```diff
@@ -38,14 +38,14 @@ const config: Config = {
           // Please change this to your repo.
           // Remove this to remove the "edit this page" links.
           editUrl:
-            'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+            'https://github.com/polymerdao/monomer/tree/main/doc/',
         },
         blog: {
           showReadingTime: true,
           // Please change this to your repo.
           // Remove this to remove the "edit this page" links.
           editUrl:
-            'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+            'https://github.com/polymerdao/monomer/tree/main/doc/',
```

Comment:
LGTM, although I didn't run locally to click generated links.

I guess that the `blog` key here isn't actively used, but good to have it set in the right direction anyhow.

Reply:
I ran it locally to double check, links are good.

> I guess that the `blog` key here isn't actively used, but good to have it set in the right direction anyhow.

This actually brings up a good point. We can disable the blog feature as a whole if we don't need it.

Reply:
Sounds good

Reply:
Cool, I'll remove the blog and then this PR should be good to merge